cardano-node-chairman/cardano-node-chairman.cabal

```diff
@@ -89,5 +89,5 @@ test-suite chairman-tests
   ghc-options:          -threaded -rtsopts -with-rtsopts=-N -with-rtsopts=-T
 
   build-tool-depends:   cardano-node:cardano-node
```

Comment:
Should we also constraint the node exec?

Reply:
Chairman is in the same repo as node. If they're built together, it will get the version from the repo. I'm not aware about any other build method.